src/protocol.rs

```diff
@@ -210,10 +210,22 @@ impl Participant {
     pub fn is_auxinfo_done(&self, auxinfo_identifier: Identifier) -> Result<bool> {
         let mut fetch = vec![];
         for participant in self.other_participant_ids.clone() {
-            fetch.push((AuxInfoStorageType::Public, auxinfo_identifier, participant));
+            fetch.push((
+                PersistentStorageType::AuxInfoPublic,
+                auxinfo_identifier,
+                participant,
+            ));
         }
-        fetch.push((AuxInfoStorageType::Public, auxinfo_identifier, self.id));
-        fetch.push((AuxInfoStorageType::Private, auxinfo_identifier, self.id));
+        fetch.push((
+            PersistentStorageType::AuxInfoPublic,
+            auxinfo_identifier,
+            self.id,
+        ));
+        fetch.push((
+            PersistentStorageType::AuxInfoPrivate,
+            auxinfo_identifier,
+            self.id,
+        ));
```

Comment:
suggestion: Can you use the `contains` notation here, too? Also, can you combine the public queries into one with `self.all_participants()`?

Reply:
Currently `Storage` doesn't have a `contains` call (I added that to `LocalStorage` only). I can add `Storage::contains` but given that `Storage` should go away I wonder if it's worth it?

Reply:
Oh no, I didn't realize this was on persistent storage. Don't bother adding!